src/templates.rs

```diff
@@ -12,14 +13,27 @@ use semver;
 
 pub use liquid::value::{Object as Parameters, Value};
 
+#[allow(clippy::all)]
 mod external {
     include!(concat!(env!("OUT_DIR"), "/_template_files.rs"));
```

Comment:
Would be great if we could test the output of the `build.rs` script. Could you add a `tests` module which checks the keys / values are what we expect?